gitbook/formatting.md

```diff
@@ -13,10 +13,10 @@ var locales = {
 }
 ```
 
-Template the following (notice the tripple brackets):
+Template the following (triple brackets are deprecated in vue 2.0 and replaced by v-html):
 
 ```html
-<p>{{{ $t('message.hello') }}}</p>
+<p v-html="$t('message.hello')"></p>
```

Comment:
`vue-i18n` is also supporting for vue 1.x
We hope you will add the example for vue 1.x :)